android/app/src/main/java/app/organicmaps/sound/MediaPlayerWrapper.java

```diff
@@ -51,11 +51,13 @@ public void release()
   }
 
   @NonNull
+  @SuppressWarnings("deprecation") // https://github.com/organicmaps/organicmaps/issues/3632
```

Comment:
Will java.util.concurrent work on Android 5-10?

Reply:
It seems to be working. I thought about importing the entire AsyncTask instead, but let's wait until it's actually removed from Android.